rules/retrieve.smk

```diff
@@ -1036,35 +1051,51 @@ if (LAU_REGIONS_DATASET := dataset_version("lau_regions"))["source"] in [
         script:
             "../scripts/retrieve_seawater_temperature.py"
 
-    rule retrieve_hera_data_test_cutout:
+
+if (HERA_TEST_CUTOUT_DATASET := dataset_version("hera_test_cutout"))["source"] in [
```

Comment:
Can you figure out how the `build`ing of the HERA cutout works, e.g. for the `be_2013-03-01_to_2013-03-8.nc` data? I'd assume it is downloading the data and then cutting its extent to match the `atlite` `cutout`?

If yes, then we can apply the same logic to the "test" cutout (`be`) and the default cutout (`europe-...`) and have a `build_hera_cutout` for `primary` and a `retrieve_hera_cutout` for `archive`.